dts/arm/st/h7rs/stm32h7rs.dtsi

```diff
@@ -497,6 +497,32 @@
 			status = "disabled";
 		};
 
+		fdcan1: can@4000a000 {
+			compatible = "st,stm32-fdcan";
```

Comment:
There is also a `st,stm32h7-fdcan` compatible. Are you sure you picked-up the right one ?